docs/terminal/eastsale.md

```diff
@@ -1,10 +1,247 @@
----
-  title: East Sale TCU
----
+# East Sale TCU
 
---8<-- "includes/abbreviations.md"
+## Positions
 
-Reserved.
+| Name               | ID      | Callsign             | Frequency        | Login Identifier    |
+| ------------------ | ------- | ------------------- | ---------------- | ------------------- |
+| **East Sale TMA**  | **ESL** | **East Sale Approach** | **123.300**      | **ES_APP**          |
```

Comment:
Can we confirm that login identifier is correct @zkgell?

Reply:
![image](https://github.com/user-attachments/assets/86bdbe3d-686d-4620-acb2-19b2eac3c9f5)